tests/ui/unpretty/mir/global-asm-const.rs

```diff
@@ -0,0 +1,13 @@
+//@ compile-flags: -Zdump-mir=all --crate-type=lib
+//@ needs-asm-support
+//@ check-pass
+
+use std::arch::{asm, global_asm};
+
+// test that pretty mir printing of `const` operands in `global_asm!` does not ICE
+
+global_asm!("/* {} */", const 5);
+
+pub fn foo() {
+    unsafe { asm!("/* {} */", const 5) };
```

Comment:
This isn't exercising anything since this doesn't ICE on master. Please remove it.